### DISCUSSION TITLE
gatsby-plugin-sitemap is not compatible with gatsby version 4.17.0 ??

### DISCUSSION THREAD

**Discussion by ghost** [gatsbyjs/gatsby#35988] [Help]
I'm trying to upgrade to gatsby version 4.17.0 and one of the warnings i'm running to says "Plugin gatsby-plugin-sitemap is not compatible with your gatsby version 4.17.0 - It requires gatsby@^3.0.0-next.0" does that mean this plugin can't be used at all with updated versions of gatsby? Is there a different plugin I have to migrate to?

**Comment by LekoArts** [comment]
Hi! You'll also want to update all gatsby-related plugins like `gatsby-plugin-sitemap` to their latest version. Once done you won't see this warning anymore. You can see all migration notes in https://www.gatsbyjs.com/docs/reference/release-notes/migrating-from-v3-to-v4/ or in the README of the individual plugin.
